scripts/install

```diff
@@ -228,6 +229,7 @@ cp ../conf/log.yaml /etc/matrix-$app/log.yaml
 
 ynh_replace_string __APP__ $app "$homeserver_config_path"
 ynh_replace_string __DOMAIN__ $domain "$homeserver_config_path"
+ynh_replace_string __SERVER_NAME__ $server_name "$homeserver_config_path"
```

Comment:
Note that you need to  upgrade also the Upgrade and config file as explained in line 222.

Reply:
Sure. Is changing the version in the manifest enough to simulate an upgrade ?
In short : how to quickly test an upgrade ?

Reply:
Just do an app upgrade.

You can update the manifest but for now it's not mandatory